### ISSUE TITLE
Create public board fails

### ISSUE THREAD

**Issue by kengoldfarb** [kengoldfarb/pin-it-node#2]
@ivebencrazy - I just started writing some basic unit tests for the new stuff and I'm seeing an error when trying to create a new board.

Check it out:
https://github.com/kengoldfarb/pin-it-node/blob/master/tests/pinItTests.js

**Comment by kengoldfarb** [comment]
@ivebencrazy So I (finally) just started writing some unit tests for the board stuff...create works great after changing to lower case.

But delete isn't working for me.  [Check out the test I'm using for delete](https://github.com/kengoldfarb/pin-it-node/blob/master/tests/pinItTests.js#L38) if you have the chance.  I've tried it with just the boardurl and with both the boardurl and boardId with no luck.

I'm going to mess with it more but figured I'd ask if you had any insight?

**Comment by ivebencrazy** [comment]
I'll look at it before the weekend.  Sorry I'm a bit slow.

**Comment by kengoldfarb** [comment]
@ivebencrazy No worries, thanks man!

**Comment by kengoldfarb** [comment]
@ivebencrazy - I just pushed a new update since Pinterest released some breaking changes.

FYI - during my testing it looks like `_getBoardId` is always failing to find the board id so I've removed that from the docs and put back in the requirement for providing `boardId`.  If you have time, check out the commented out unit test that tries to use the board url instead.

However, creating and deleting of boards and deleting of pins is working (with boardId) so I added those to the latest release
